core/runtime/src/main/java/io/quarkiverse/langchain4j/runtime/devui/EmbeddingStoreJsonRPCService.java

```diff
@@ -20,9 +23,17 @@ public class EmbeddingStoreJsonRPCService {
 
     EmbeddingModel embeddingModel;
 
+    private static Logger log = Logger.getLogger(EmbeddingStoreJsonRPCService.class);
+
     public EmbeddingStoreJsonRPCService() {
         // if there are more models/stores, try to choose the default
-        embeddingModel = CDI.current().select(EmbeddingModel.class, new Default.Literal()).get();
+        try {
+            embeddingModel = CDI.current().select(EmbeddingModel.class, new Default.Literal()).get();
+        } catch (UnsatisfiedResolutionException ex) {
+            log.info("");
```

Comment:
I have zero idea why, but if I remove this, the log doesn't appear (or gets garbled up together with the log saying "powered by Quarkus...") @geoand any idea?

Reply:
We could also simply remove these logging messages and only show errors on the dev UI page.

Reply:
I removed it in the end because I wasn't able to reproduce it anymore today. Probably some weirdly compiled Quarkus...